### ISSUE TITLE
Support for reading the windows events from the  Linux host

### ISSUE THREAD

**Issue by kamaruddin-desai** [dotnet/runtime#31576]
I was happy that when I heard the .NET Core can be used to build apps onto the across the platforms.
Specifically the Linux one. I've code snippet which reads the windows event from the windows localhost or the remote windows host. For that purpose, I've created the below issue and the code snippet that I am using can be seen here[https://github.com/dotnet/corefx/issues/40925](https://github.com/dotnet/corefx/issues/40925).

Thi issue focus on the milestone to 

> **Support for reading the windows events from the  Linux host**

**Comment by jkotas** [comment]
> Another direction, more likely to help you, is to write the code yourself and share it on nuget.

I agree that it would be best to do this as independent nuget package.

You can find number of posts on the Internet that discuss this (e.g. https://stackoverflow.com/questions/5733687/querying-windows-event-logs-remotely-linux ).

One way to do this may be implement https://github.com/dotnet/runtime/issues/23918 first and then read the eventlog from Linux using WMI.
